docs/content/getting-started.md

```diff
@@ -181,10 +183,10 @@ this:
 
 ## Modules Calling Modules
 
-This is all well and good, but we don't want to have to write things the same
-*every time* right? Well, good news: that's what modules are for! Now that
-you've written a module, you can require it from any other module to add it to
-your tree. Create a new module, let's call it `helloYou.hcl`:
+Let's look at how we can take advantage of reusability. Good news: that's what
+modules are for! Now that you've written a module, you can require it from any
+other module to add it to your tree. Create a new module, let's call it
```

Comment:
We could lose "Create a new module, let's call it `helloWorld.hcl`." in favor of the imperative sentence following it.